content/pt-br/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,266 @@
+---
+reviewers:
+- erictune
+title: Init Contêineres
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Essa página provê uma visão geral de init contêineres: especializada nos contêineres que executam antes de app contêineres em um {{< glossary_tooltip text="Pod" term_id="pod" >}}. Init contêineres podem conter utilidades ou configurações que não estão presentes em uma imagem de um app.
+
+Você pode especificar init contêineres em uma especificação de Pod ao longo de um array de `containers` (que descreve app contêineres).
+
+<!-- body -->
+
+## Entendendo init contêineres
+
+Um {{< glossary_tooltip text="Pod" term_id="pod" >}} pode ter multiplos contêineres executando apps com ele, mas ele também pode ter um ou mais init contêineres, que são executados antes dos app contêineres serem iniciados.
```

Comment:
Some points in this paragraph:
* We usually use Pod with a capital P in Portuguese translations, because it denotes a Kubernetes object.
* "running apps within it" would be better translated with a sentence that gives the sense of something contained inside the Pod. Suggestion: "um Pod pode ter múltiplos contêineres executando aplicações nele".
* Missing accent in "múltiplos".